src/FlaUI.WebDriver/app.manifest

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Could you check if this file is copied to the publish directory?

Reply:
I believe that it's embedded into the generated dll, but I need to confirm that.

Reply:
Confirmed that the published executable runs in DPI aware mode. If you want to check it yourself, you can [see a process' DPI awareness from Task Manager](https://www.tenforums.com/tutorials/120082-see-dpi-awareness-running-apps-task-manager-windows-10-a.html). I couldn't take a screenshot because apparently you can't take a screenshot of Task Manager. I didn't know that.

Reply:
OK, let's see if it works after building and releasing with GitHub...